tests/input/localicipe.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 # Strings that are used in Android only
-ANDROID:
+COLLECTION-A:
```

Comment:
I've decided to rename the root level props to make it more clear that output-types + collections are totally different concepts now. WDYT?

Reply:
I agree that it's a bit confusing to have android in both outputTypes and collections. The idea is to have one collection for the react-native project, right? otherwise, we will get strings that won't be used in android and iOS